code/game/atoms.dm

```diff
@@ -1376,6 +1376,8 @@
 /// Helper for logging chat messages or other logs with arbitrary inputs (e.g. announcements)
 /atom/proc/log_talk(message, message_type, tag=null, log_globally=TRUE, forced_by=null)
 	var/prefix = tag ? "([tag]) " : ""
+	if(message_type == LOG_WHISPER)
+		prefix += "whispers "
```

Comment:
Undocumented change?

Reply:
If you mean DMDoc what should the documentation be? It's an existing argument of the proc I simply implemented for this purpose. I mentioned the purpose of it in the PR text.

Reply:
I meant as in whispers weren't marked as such until this PR, and it is not in the changelog.

Reply:
I'll add that in the changelog then, thanks.

Reply:
This was documented in the body of the PR ( `Also in this PR, I made whispers visually distinguishable from regular say's.` ) and the change is even showcased among the images.
Please read it before calling it undocumented.
Asking for it to be changeloged is fine, but saying it's an undocumented change is misleading at best.

Reply:
Missed that bit, my bad.